components/WeeklyPickButton/WeeklyPickButton.tsx

```diff
@@ -65,4 +65,4 @@ const WeeklyPickButton: React.FC<WeeklyPickButtonProps> = ({
   );
 };
 
-export { WeeklyPickButton };
+export { WeeklyPickButton }
```

Comment:
This change seems out of scope considering we're only looking at the signup page, please reverse this change.